content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -124,7 +124,8 @@ different Kubernetes components.
 | `IndexedJob` | `false` | Alpha | 1.21 | 1.21 |
 | `IndexedJob` | `true` | Beta | 1.22 | |
 | `IngressClassNamespacedParams` | `false` | Alpha | 1.21 | 1.21 |
-| `IngressClassNamespacedParams` | `true` | Beta | 1.22 | |
+| `IngressClassNamespacedParams` | `true` | Beta | 1.22 | 1.22 |
+| `IngressClassNamespacedParams` | `true` | GA | 1.23 | - |
```

Comment:
I think this needs to move down to the "Feature gates for graduated or deprecated features" section.

Reply:
@robscott updated.